src/python/gem5/resources/client_api/client_query.py

```diff
@@ -0,0 +1,22 @@
+from typing import Optional
```

Comment:
This file has to have a copyright statement.

Reply:
@Harshil2107 , you resolved this conversation without addressing the comment.